packages/react-reconciler/src/ReactFiberLane.js

```diff
@@ -299,8 +299,8 @@ export function getNextLanes(root: FiberRoot, wipLanes: Lanes): Lanes {
   // time it takes to show the final state, which is what they are actually
   // waiting for.
   //
-  // For those exceptions where entanglement is semantically important, like
-  // useMutableSource, we should ensure that there is no partial work at the
```

Comment:
@acdlite are there remaining cases of this? Maybe the replacement, `useSyncExternalStore` or can this block be removed?